dask/bag/tests/test_random.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+import dask.bag as db
+from dask.bag import random
+
+
+def test_choices_size():
+    """
+    Number of randomly sampled elements are exactly k.
+    """
+    a = db.from_sequence(range(20), npartitions=3)
+    s = random.choices(a, k=2)
+    assert len(list(s.compute())) == 2
+
+
+def test_choices_size_over():
+    """
+    Number of randomly sampled are more than the elements.
+    """
+    a = db.from_sequence(range(3), npartitions=3)
+    s = random.choices(a, k=4)
+    assert len(list(s.compute())) == 4
+
+
+def test_choices_size_over_repartition():
+    """
+    Number of randomly sampled are more than the elements on each partition.
+    """
+    a = db.from_sequence(range(10), partition_size=9)
+    a = a.repartition(3)
+    s = random.choices(a, k=2)
+    assert len(list(s.compute())) == 2
```

Comment:
I am not sure what this is meant to test.

Reply:
This test assures that the function works also when there are empty partitions.
I am adding an additional assert, so is clear how many elements are on each partition after the execution of the repartition method.

Reply:
Ok thanks! The extra assert helps :)